core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -555,6 +557,30 @@ public final void setTlsEnabled(final boolean enabled) {
         this.tlsEnabled = enabled;
     }
 
+    /**
+     * Get the host override for TLS SNI Server Name set by {@link #setSniServerName(String)}.
```

Comment:
active present tense, no need to refer to the setter (this should be clear by means of the property name)